.yarnrc.yml

```diff
@@ -1,3 +1,2 @@
 enableGlobalCache: true
 nodeLinker: node-modules
-checksumBehavior: ignore
```

Comment:
I haven't used a recent version of `yarn`, but I couldn't even run `yarn install` without this